docs/preview/guidance/use-with-dotnet-and-functions.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Using Arcus & Serilog in .NET Core and/or Azure Functions"
+layout: default
+---
+
+# Using Arcus & Serilog in .NET Core and/or Azure Functions
+
+When using Arcus Observability & Serilog it is mandatory that your application is configured correctly to ensure everything works smoothly and all features are working fine.
+
+We encourage you to follow the standard [Serilog instructions](https://github.com/serilog/serilog-aspnetcore#instructions) on setting your application up. 
+
+Some aspects we would like to highlight are:
+
+- Make sure to call [`UseSerilog`](https://www.nuget.org/packages/Serilog.AspNetCore) when creating a `IHostBuilder`
+- Remove default for logging including its configuration in `appsettings.json` *(if applicable)*
+
+If you cannot use `UseSerilog`, you can still configure it by using `AddSerilog` as a logging provider; but we recommend removing all other providers with `loggingBuilder.ClearProviders()` so that they don't interfer.
+
+## Setting up Serilog with Azure Functions
+
+Using Serilog with Azure Functions requires some guidance and we've made it a bit easier to use.
+
+Before we get started, install our NuGet package for Azure Functions:
+
+```
+PM > TODO @stijnmoreels
+```
+
+Once that is done, you can configure Serilog during startup as following:
+
+```csharp
+using Arcus.TODO.@stijnmoreels;
```

Comment:
It's an complete example, I'm jus wondering if it's not "too much". Maybe having a little less enrichers levels, ... IDK. It seems a bit "noise" for what we want to talk about here?

Reply:
Or maybe with a code comment to highlight the parts is also an option.

Reply:
That's fine for me, do you want to get rid of the code and use // Your logic here?

Reply:
Was just a suggestion. Not sure how and what lines we could remove to keep it as simple as possible. Maybe the enrichers can be removed?

Reply:
I wanted to show a complete setup but how it is being used in the function is less relevant?

Reply:
It's okay for me, I'll resolve this. A complete setup exam will be ok!